pkg/cmd/cli/cmd/login.go

```diff
@@ -178,6 +183,43 @@ func (o LoginOptions) Validate(args []string, serverFlag string) error {
 	return nil
 }
 
+func (o LoginOptions) checkConfigIsWritable(f *osclientcmd.Factory) (bool, error) {
```

Comment:
If the file doesn't exist, just short this out to `return true` for now.  That will be al ess common case.